Models/SwissBitcoinPayInvoiceModel.cs

```diff
@@ -0,0 +1,18 @@
+﻿
+namespace SmartStore.SwissBitcoinPay.Models
+{
+
+    public struct SwissBitcoinPayInvoiceModel
+    {
+        public string title;
+        public string description;
+        public decimal amount;
+        public string unit;
+        //public bool onChain;
+        //public int delay;

```

Comment:
You can delete this comment, and set delay to `60` for all created invoices (60 minutes)

Reply:
ok, done.